apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -92,7 +110,16 @@ export class PackageChangeAnalyzer {
         | RushConfigurationProject
         | undefined = this._rushConfiguration.findProjectForPosixRelativePath(filePath);
       if (owningProject) {
-        projectHashDeps.get(owningProject.packageName)!.set(filePath, fileHash);
+        const relativePath: string = path.relative(owningProject.projectRelativeFolder, filePath);
```

Comment:
I'd recommend just slicing off the prefix, since `owningProject` will only be defined if filePath literally starts with `projectRelativeFolder`

Reply:
Also, it keeps all paths normalized to POSIX slashes (which git returns, and which projectRelativeFolder has been normalized to).